rpc/src/v1/traits/parity_accounts.rs

```diff
@@ -72,57 +72,6 @@ build_rpc_trait! {
 		#[rpc(name = "parity_setAccountMeta")]
 		fn set_account_meta(&self, H160, String) -> Result<bool>;
 
-		/// Sets addresses exposed for particular dapp.
-		/// Setting a non-empty list will also override default account.
-		/// Setting `None` will resets visible account to what's visible for new dapps
-		/// (does not affect default account though)
-		#[rpc(name = "parity_setDappAddresses")]
-		fn set_dapp_addresses(&self, DappId, Option<Vec<H160>>) -> Result<bool>;
```

Comment:
I wonder if we should mark the methods as deprecated first and only remove them in the next version?

Reply:
cc @5chdn. But I think we have 2.0 so it's okay to break backward compatibility from there? ;)

Reply:
Let's remove them for 2.0.1 and highlight that in release notes.